prody/proteins/interactions.py

```diff
@@ -867,14 +865,31 @@ def calcProteinInteractions(atoms, **kwargs):
     return AllInteractions
 
 
-def calcHydrogenBondsDCD(atoms, trajectory, distA=3.0, angle=40, cutoff_dist=20, **kwargs):   
+def calcHydrogenBondsDCD(atoms, trajectory=None, distA=3.0, angle=40, cutoff_dist=20, **kwargs):   
```

Comment:
you should probably rename the function if it doesn't need a trajectory. You could even have all the functions as general ones that can take a trajectory or not.

Reply:
'trajectory' is for dcd files only. If you use pdb file with multiple pdb frames, then the function will compute it all as Ensemble.

Reply:
but, trajectory can be None and it can use multi-frame pdb files. 

(It actually can't use multiple pdb files without *atoms and buildPDBEnsemble, which I'm sure you don't want)

You could call it calcHydrogenBondsTrajectory because it can use multi-frame Atomic not just dcd Trajectory.